### DISCUSSION TITLE
Wrong translation in your blog

### DISCUSSION THREAD

**Discussion by ooghry** [community/community#14385] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello,

In this [post](https://github.blog/2021-01-05-advancing-developer-freedom-github-is-fully-available-in-iran/), you've had translated `Syria` as `روسیه(Russia)`, but the correct name is `سوریه`.

**Comment by martinwoodward** [comment, marked as answer]
خیلی ممنون!

Fixed - thank you very much.

**Reply by 133489** [reply]
السلام

**Reply by 133489** [reply]
كيفية سحب الارباح
